akvo/templates/project_directory.html

```diff
@@ -264,6 +271,26 @@ <h4 class="detailedInfo">{% trans "Finance" %}</h4>
         }
     </script>
 
+    {# More partners data #}
+    {% for project in page %}
+    <script type="application/json" id="more-partners-{{project.id}}">
```

Comment:
The whole script tag should be copied into project_main.html since the partner tooltip is used there too!